### ISSUE TITLE
Dynamic Content Support For Number Fields - Counters, Review stars, etc...

### ISSUE THREAD

**Issue by reachadam** [elementor/elementor#8347]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Elementor Pro has dynamic field support however its only for text-based fields and not fields that only take numbers. An example is the review starts, or counters, etc. 

**Describe the solution you'd like**
Dynamic field support for these fields

**Describe alternatives you've considered**
Only alternative is Beaver Themer which supported this from day one. 

**Additional context**
None that I can think of

**Comment by shilo-ey** [comment]
Hi @reachadam 

This is actually something that we are working on and should be released in the next versions. 

For now, you can use the provided filters and change the control type form number to text and activate dynamic capabilities on it, ex:
```PHP
add_action( 'elementor/element/star-rating/section_rating/before_section_end', function( $control_stack, $args ) {
	// Get existing control
	$control = \Elementor\Plugin::instance()->controls_manager->get_control_from_stack( $control_stack->get_unique_name(), 'rating' );

	if ( is_wp_error( $control ) ) {
	    return;
    }

	$control['type'] = 'text';
	$control['dynamic'] = [
		'active' => true,
	];
	// Update the control
	$control_stack->update_control( 'rating', $control );
}, 10, 2 );
```

In the meantime, I merged this issue with #5952, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!

**Comment by reachadam** [comment]
Thank you @shilo-ey 

Is there any way to get this to work with fields that are numbers? My custom field is a number, and it's not showing any of the number fields to link up.
